crates/solver/src/solver/http_solver.rs

```diff
@@ -399,6 +399,10 @@ impl Solver for HttpSolver {
                     let (model, context) = self
                         .prepare_model(id, orders, liquidity, gas_price, external_prices)
                         .await?;
+                    tracing::debug!(
+                        "Problem sent to http solvers (json):\n{}",
+                        serde_json::to_string_pretty(&model).unwrap()
```

Comment:
Do you think it makes sense to re-write this log as:
```suggestion
                        json =% serde_json::to_string(&model).unwrap(),
                        "Problem sent to http solvers"
```

And something similar for the log below?

The reason is that newlines break search-ability with tools like `fzf`, `grep`, et al. when using `kubeclt logs`.

Reply:
Actually we have been doing this in quasimodo for a long time, and it works better if we include new lines. The reason is that when copy-pasting from the logs to a file, lines are sometimes broken at bad places (like in the middle of a string). And for quasimodo the logger never got confused (I bet it looks for some prefix like the time to assume a new "log" line).

Reply:
>  it works better

I suppose I should add "for us" because we simply copy it from kibana. I guess you're using tools to get it programatically from `kubectl logs` ?